qa/TL0_python-self-test/test_body.sh

```diff
@@ -1,7 +1,7 @@
 #!/bin/bash -e
 
 test_nose() {
-    for test_script in $(ls test_operator_*.py test_pipeline*.py test_backend_impl.py); do
+    for test_script in $(ls test_operator_*.py test_pipeline*.py test_backend_impl.py test_RNNT_data_pipeline.py); do
```

Comment:
I suggest that you call your file `test_pipeline_rnnt.py` so that it will picked by the expression `test_pipeline*.py`

Reply:
Done